doc/reference/reference_lua/box_stat/net.rst

```diff
@@ -15,27 +15,50 @@ box.stat.net()
           second in the last 5 seconds
         * ``SENT.total`` and ``RECEIVED.total`` -- total number of bytes sent/received
           since the server started
-        * ``CONNECTIONS.rps`` -- number of connections opened per second in the last 5 seconds
+        * ``CONNECTIONS.current`` -- number of open connections
+        * ``CONNECTIONS.rps`` -- average number of connections opened per second in the last 5 seconds
         * ``CONNECTIONS.total`` -- total number of connections opened since the server started
         * ``REQUESTS.current`` -- number of requests in progress, which can be
           limited by :ref:`box.cfg.net_msg_max <cfg_networking-net_msg_max>`
-        * ``REQUESTS.rps`` -- number of requests processed per second in the last 5 seconds
-        * ``REQUESTS.total`` -- total number of requests processed since startup
+        * ``REQUESTS.rps`` -- average number of requests processed per second in the last 5 seconds
+        * ``REQUESTS.total`` -- total number of requests processed since the server started
+        * ``REQUESTS_IN_PROGRESS.current`` -- number of requests being currently processed by the :ref:`TX thread <memtx-memory>`
+        * ``REQUESTS_IN_PROGRESS.rps`` -- average number of requests processed by the TX thread per second in the last 5 seconds
+        * ``REQUESTS_IN_PROGRESS.total`` -- total number of requests processed by the TX thread since the server started
+        * ``STREAMS.current`` -- number of active :doc:`streams </book/box/stream>`
+        * ``STREAMS.rps`` -- average number of streams opened per second in the last 5 seconds
+        * ``STREAMS.total`` -- total number of streams opened since the server started
+        * ``REQUESTS_IN_STREAM_QUEUE.current`` -- number requests waiting in the streams' queues
+        * ``REQUESTS_IN_STREAM_QUEUE.rps`` -- average number of requests in the streams' queues per second in the last 5 seconds
+        * ``REQUESTS_IN_STREAM_QUEUE.total`` -- total number of requests placed in the streams' queues since the server started
```

Comment:
```suggestion
        * ``REQUESTS_IN_STREAM_QUEUE.current`` -- number of requests waiting in stream queues
        * ``REQUESTS_IN_STREAM_QUEUE.rps`` -- average number of requests in stream queues per second in the last 5 seconds
        * ``REQUESTS_IN_STREAM_QUEUE.total`` -- total number of requests placed in stream queues since the server started
```
Possessive with inanimate objects looks non-optimal.